cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/notifier/SlackSelfHealingNotifier.java

```diff
@@ -66,13 +66,13 @@ public void alert(Object anomaly, boolean autoFixTriggered, long selfHealingStar
             return;
         }
 
-        String text;
+        String text =  String.format("%s detected %s. Self healing %s.", anomalyType, anomaly,
+                _selfHealingEnabled.get(anomalyType) ? String.format("start time %s", toDateString(selfHealingStartTime)) : "is disabled");
+
         if (autoFixTriggered) {
-            text = "Self-healing has been triggered.";
-        } else {
-            text = String.format("%s detected %s. Self healing %s.", anomalyType, anomaly,
-                    _selfHealingEnabled.get(anomalyType) ? String.format("start time %s", toDateString(selfHealingStartTime)) : "is disabled");
+            text = text + "\n Self-healing has been triggered.";
```

Comment:
Is it possible to
  1. Avoid having an extra space after new line, and
  2. Prefer `%n` rather than `\n`. -- please see https://stackoverflow.com/questions/1883345/whats-up-with-javas-n-in-printf

Maybe something like the following?
```
        String text = String.format("%s detected %s. Self healing %s.%s", anomalyType, anomaly,
                                    _selfHealingEnabled.get(anomalyType) ? String.format("start time %s", toDateString(selfHealingStartTime))
                                                                         : "is disabled",
                                    autoFixTriggered ? "%nSelf-healing has been triggered." : "");
```

Reply:
Great suggestion, thanks!

Addressed in most recent commit.